core/src/prim.rs

```diff
@@ -211,21 +211,18 @@ impl Op {
       Self::U16(op) => op.apply2(x, y),
       Self::U32(op) => op.apply2(x, y),
       Self::U64(op) => op.apply2(x, y),
-      Self::U128(op) => op.apply2(x, y),
+      // TODO These break wasm and wasm-pack
+      // Self::U128(op) => op.apply2(x, y),
```

Comment:
Can we comment out the U128 and I128 variants on the `Literal` and `Op` enums as well? I think this branch should include a complete removal of those features